apps/mdn/utils/efs_backup/README.md

```diff
@@ -0,0 +1,80 @@
+# MDN EFS <--> S3 Backup/Restore
```

Comment:
This is fine. I'd also accept ↔ or other [unicode double arrows](https://unicode-table.com/en/sets/arrows-symbols/)